In [1]:
import numpy as np # linear algebra 
import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)
# stats models: regression fitting via formulas
import statsmodels.formula.api as smf
# stats models: regression fitting via matrices of regression design
import statsmodels.api as sm

In [3]:
df = pd.read_csv('https://raw.githubusercontent.com/artamonoff/Econometrica/master/python-notebooks/data-csv/wage2.csv')
# Описательные статистики по переменным модели
df[['IQ','age']].describe()

IQ         age
count  935.000000  935.000000
mean   101.282353   33.080214
std     15.052636    3.107803
min     50.000000   28.000000
25%     92.000000   30.000000
50%    102.000000   33.000000
75%    112.000000   36.000000
max    145.000000   38.000000

In [4]:
# специфицируем моделл через формулу
wage_eq1 = smf.ols(formula='np.log(wage)~IQ+age', data=df).fit()
# Коэфициенты модели с округление
wage_eq1.params.round(3)

Intercept    5.077
IQ           0.009
age          0.024
dtype: float64

### Спецификация 

$$
log(wage)= 5.077+0.009*IQ+0.024*age
$$

### Интерпретация 
1. При увеличении значения IQ теста на 1, зарплата одного сотрудника в тысячах евро изменится на 0.9% при прочих равных условиях.
2. При увеличении возраста сотрудника на 1 год, его зп уведичится на 2.4% при прочих равных условиях.